data/hai/hai missions.txt

```diff
@@ -1062,6 +1062,362 @@ mission "Hai Honeymoon"
 		payment 100000
 		dialog `Instead of landing in the spaceport where they can be seen, you drop Anaya and Touhar off as close as you can to where they will be staying. The house is a few kilometers away, but they assure you that they can make it on their own. They pay you <payment> and thank you for bringing them this far.`
 
+mission "Hai Solifuge response"
+	landing
+	invisible
+	to offer
+		has "Wanderers Solifuge Recon 2: done"
+	on offer
+		event "Hai Pond Striders released" 10
+		fail
+
+event "Hai Pond Striders released"
+	fleet "Large Hai"
+		add variant 1
+			"Hai Pond Strider" 2
+			"Flea (Pulse)" 8
+			"Flea" 8
+			"Lightning Bug" 2
+		add variant 1
+			"Hai Pond Strider" 3
+			"Flea (Pulse)" 12
+			"Flea" 12
+		add variant 1
+			"Hai Pond Strider"
+			"Flea" 8
+			"Geocoris"
+			"Aphid (Anti-Missile)" 2
+			"Grasshopper"
+		add variant 1
+			"Hai Pond Strider" 2
+			"Flea (Pulse)" 8
+			"Flea" 8
+			"Centipede"
+
+mission "Strider: View of the Pond"
+	description "Deliver a message regarding new Unfettered ships from Ambassador Sayari to Hai elders on <destination>."
+	name "Message Hai-home"
+	landing
+	source "Vara K'chrai"
+	destination "Hai-home"
+	to offer
+		has "Wanderers Solifuge Recon 2: done"
+	on offer
+		conversation
+			`"Ambassador Sayari has a message for the Hai elders about the Solifuge, and needs your help to deliver it to <destination>."`
```

Comment:
She does? Isn't she a Hai? Why would she need your help? This message is kinda too short and could bear expansion.